### ISSUE TITLE
From the splat view you can't tell what events a device reports

### ISSUE THREAD

**Issue by dhylands** [mozilla-iot/gateway#1415]
Currently, the only way to tell what events a device generates is to try to create a rule using that device. You can then see the events that the device is able to generate.

I think that we should be able to see these somehow from within the splat (thing detail) view as well.

**Comment by benfrancis** [comment]
I understand what you mean, but I'm not sure adding a bubble to the detail view for every supported event which just displays the name of the event is necessarily useful UI. That's why I'm trying to reframe the requirement in terms of what task the user might actually be trying to achieve.

**Comment by dhylands** [comment]
I don't think we need a bubble for every event. Perhaps one bubble if there are any events, and if you click the bubble it expands to a list of the events along with their associated descriptions. Having a counter associated with each event would also be useful as a way of discovering what actions produce what events.

**Comment by benfrancis** [comment]
Again, what is the use case? If a device generates 100 events a second, a counter is not going to be very helpful. If a device has 4 different types of click event and 5 error conditions, what use does the end user have for a list of that information?

You must have been trying to do something when you came across this need. Was it a user need (e.g. "As a user I want to find out how many times my button has been pressed in the last week") or a developer need ("As a developer I want to check that the event my adapter is firing triggers a `event` message in the Web Thing WebSocket API")?

**Comment by dhylands** [comment]
I'm trying to verify that my device is working properly, and figure out how to interact with it using the gateway. I notice that it does flash up a banner at the bottom for each event that's generated, and I didn't see that before.

Right now the only place I can see what events the device generates is in the rules engine which doesn't appear obvious to me.

**Comment by dhylands** [comment]
> Again, what is the use case? If a device generates 100 events a second, a counter is not going to be very helpful.

They'll be able to tell that the device is working.

> If a device has 4 different types of click event and 5 error conditions, what use does the end user have for a list of that information?

They'll know, whether the device supports doublePress or not, for example.
